policy/modules/contrib/firewalld.te

```diff
@@ -69,6 +69,7 @@ manage_dirs_pattern(firewalld_t, firewalld_var_run_t, firewalld_var_run_t)
 files_pid_filetrans(firewalld_t, firewalld_var_run_t, { file dir })
 can_exec(firewalld_t, firewalld_var_run_t)
 
+kernel_load_module(firewalld_t)
```

Comment:
Just a nit, but it would be nicer to move this line close to the `kernel_request_load_module(firewalld_t)` a few lines below.

Reply:
Right, but it is against the alphabetical-order rule. One can never satisfy everybody.